### ISSUE TITLE
Schedule metabox does not display when HPOS is enabled

### ISSUE THREAD

**Issue by Jinksi** [Automattic/woocommerce-subscriptions-core#293]
## Describe the bug

Related to #223.

The Schedule metabox does not render when HPOS is enabled, and syncing is disabled.

This has an additional side-effect of throwing the following error when updating a Subscription to status: `on-hold`: see https://github.com/Automattic/woocommerce-subscriptions-core/pull/282#issuecomment-1322434040.

```
Invalid data. First parameter was empty when passed to update_dates().
```

## To Reproduce

1. Enable [HPOS](https://github.com/woocommerce/woocommerce/wiki/High-Performance-Order-Storage-Upgrade-Recipe-Book#how-to-enable-hpos) and disable syncing (see settings screenshot).
2. Purchase a subscription via BACS or WCPay. Note the Order ID and Subscription ID.
3. View the subscription. Currently, Subscriptions are not visible via the List screen (see #223) with HPOS enabled and syncing off. View the screen via the URL `/wp-admin/admin.php?page=wc-orders&action=edit&id={{subcription_id}}`
4. Notice that the **Schedule** metabox is not visible.

<img width="300" alt="image" src="https://user-images.githubusercontent.com/3147296/203188757-208865a5-7d8d-4230-8e91-17dcd52bba7b.png">

### Expected behavior

![Screenshot 2022-11-22 at 10-07-29 Edit Subscription “Subscription” ‹ subs-core-hpos — WordPress](https://user-images.githubusercontent.com/3147296/203189519-d8c48e03-debf-4c3a-884e-f1d04b129717.png)

The Schedule metabox should be visible on the Edit Subscription screen when HPOS is enabled and syncing disabled.

### Actual behavior

The Schedule metabox is not visible on the Edit Subscription screen when HPOS is enabled and syncing disabled.

![Screenshot 2022-11-22 at 10-06-53 Orders ‹ subs-core-hpos — WordPress](https://user-images.githubusercontent.com/3147296/203189525-4864bf64-0aad-4767-9700-702f7d7e35f7.png)

## Product impact

- [x] Does this issue affect WooCommerce Subscriptions? yes
- [x] Does this issue affect WooCommerce Payments? yes

## Additional context

Discovered in https://github.com/Automattic/woocommerce-subscriptions-core/pull/282#issuecomment-1322434040.

Method that outputs metabox:
https://github.com/Automattic/woocommerce-subscriptions-core/blob/b65106656cc0c44167da6e21af9277655eabf186/includes/admin/meta-boxes/class-wcs-meta-box-schedule.php#L19-L30

`add_meta_box` called here:
https://github.com/Automattic/woocommerce-subscriptions-core/blob/b65106656cc0c44167da6e21af9277655eabf186/includes/admin/class-wcs-admin-meta-boxes.php#L80

**Comment by brucealdridge** [comment]
Related to #299

**Comment by brucealdridge** [comment]
I'm going to close this as its a duplicate,

**Comment by haszari** [comment]
@bruce.aldridge@automattic.com can  you reopen this issue or fix this up (discuss with squad). This issue has been story pointed, but is now closed as a duplicate. This issue was opened before #299 (so is the earlier report), and has more info about broken flows & intended outcome (#299 looks like a specific symptom/subset). FYI @mattallan

**Comment by brucealdridge** [comment]
> can you reopen this issue or fix this up

I've reopened the issue and linked it to the open PR #297 
I closed due to it being a duplicate of #223 rather than #299 but that wasn't super clear. Either way I have reopened.
